crates/tikv-rust-client-wasmtime/src/lib.rs

```diff
@@ -0,0 +1,75 @@
+//! Impement the WASI cache interface using a TiKV client.
+//! This is using a Wasmtime host implementation.
+
+use std::sync::Arc;
+use tokio::runtime::Runtime;
+use lazy_static::lazy_static;
+
+use wasi_cache::*;
+
+pub use wasi_cache::add_to_linker;
+
+wit_bindgen_wasmtime::export!("wit/ephemeral/wasi-cache.wit");
+lazy_static! {
+    pub(crate) static ref RUNTIME: Runtime = Runtime::new().unwrap();
+}
+/// TiKV Rust Client implementation for the WASI cache interface.
+#[derive(Clone)]
+pub struct TikvClient {
+    /// pd-endpoint for the TiKV client.
+    pub pd_endpoint: String,
+
+    /// the Tikv rust raw client (no transaction)
```

Comment:
nit: capital "The".

Reply:
Also, what does "no transaction" mean in this context?

Reply:
TiKV clients have two modes, raw and transaction. Raw client exposes `get()` and `set()` APIs that directly work with the key-value pair. The transaction client exposes APIs that support transaction, like `start()` and `submit()`